### ISSUE TITLE
Default avatar has no alt text

### ISSUE THREAD

**Issue by matthewbienz** [vanilla/vanilla#4299]
Default avatar doesn't have any alt text on the profile page

Members using assistive technology, should be able to know when the default image is being used on a page.
